source/use-cases/sensitive-data-encryption.txt

```diff
@@ -40,9 +40,42 @@ Procedure
 Requirements
 ~~~~~~~~~~~~
 
-- Point to setup and install directions
-- Nullam imperdiet lorem vitae vulputate lacinia.
-- Donec eget velit tincidunt, gravida diam ac, efficitur lacus.
+MongoDB Server 4.2 Enterprise
+  * For installation instructions, refer to the 
+    `Enterprise Edition Installation Tutorials <https://docs.mongodb.com/manual/installation/#mongodb-enterprise-edition-installation-tutorials>`_.
+
+
+MongoDB Driver Compatible with CSFLE 
+  * For a list of drivers that support CSFLE, refer to the
+    `driver compatibility table <https://docs.mongodb.com/manual/core/security-client-side-encryption/#driver-compatibility-table>`_.
+
+
+AWS Key Management Service (KMS)
+  * AWS KMS is required for the secure deployment of CSFLE.
+
+  * Compatible drivers currently include integration with AWS KMS. Support for
+    other KMS providers may be added in the future.
+
+  * The client application will need AWS KMS permissions to the GenerateDataKey,
+    Encrypt, and Decrypt actions.
+ 
+
+MongoDB Authorization Privileges
+  * In order to create data encryption keys, the client needs the
+    `ReadWrite <https://docs.mongodb.com/manual/reference/built-in-roles/#readWrite>`_ 
+    built-in role on the key vault collection which is specified in the client 
+    configuration.
+
+  * If no additional data encryption keys need to be created, the client can 
+    be downgraded to the `listCollections <https://docs.mongodb.com/manual/reference/privilege-actions/#listCollections>`_
+    privilege on the key vault collection.
+
+
+Server or VM Permissions
```

Comment:
Uncertain whether this is conventional terminology as well as whether it is clear. The next closest term I could think of is Client-Host. Any suggestions?

Reply:
How about **File System Permissions**?

Reply:
Works for me, thanks.